documentation/Apple Unified Logging and Activity Tracing formats.asciidoc

```diff
@@ -250,23 +249,21 @@ The Catalog chunk is variable of size and consists of:
 [cols="1,1,1,5",options="header"]
 |===
 | Offset | Size | Value | Description
-4+| _Chunk header (tracev3_chunk_header)_
+4+| _Chunk header (tracev3_chunk_preamble)_
 | 0 | 4 | 0x600b | Chunk tag (tag)
 | 4 | 4 | | Chunk sub tag (subtag)
-| 8 | 4 | | Chunk data size (length)
-| 12 | 4 | | [yellow-background]*Unknown (empty values?)*
-4+| _Chunk data_
+| 8 | 8 | | Chunk data size (length)
+4+| _Chunk data (tracev3_chunk_catalog_v2)_
 | 16 | 2 | | Offset of the catalog sub system strings +
 The offset is relative to the start of the catalog UUIDs
 | 18 | 2 | | Offset of the catalog process information entries +
 The offset is relative to the start of the catalog UUIDs
 | 20 | 2 | | Number of process information entries +
 | 22 | 2 | | Offset of the catalog sub chunks +
 The offset is relative to the start of the catalog UUIDs
-| 24 | 4 | | Number of sub chunks
-| 28 | 4 | | [yellow-background]*Unknown*
-| 32 | 4 | | [yellow-background]*Unknown (earliest firehose timestamp?)*
-| 36 | 4 | | [yellow-background]*Unknown*
+| 24 | 2 | | Number of sub chunks
+| 26 | 6 | | Reserved/Padding
+| 32 | 8 | | Earliest firehose timestamp
```

Comment:
Is this a wall clock timestamp or a continuous time stamp?

Reply:
This one is a continuous time value.